storage/innobase/xtrabackup/src/redo_log.cc

```diff
@@ -935,14 +937,17 @@ bool Redo_Log_Data_Manager::init() {
 
   thread = os_thread_create(PFS_NOT_INSTRUMENTED, 0, [this] { copy_func(); });
 
-  if (!log_sys_init(srv_n_log_files, srv_log_file_size,
-                    dict_sys_t::s_log_space_first_id)) {
+  lsn_t flushed_lsn = LOG_START_LSN + LOG_BLOCK_HDR_SIZE;
+
+  if (!log_sys_init(false, flushed_lsn, flushed_lsn)) {
     return (false);
   }
 
   recv_sys_create();
   recv_sys_init();
 
+  /*XB30 srv_n_log_files */
+  ulong srv_n_log_files = 2;
   ut_a(srv_n_log_files > 0);
```

Comment:
server guarantees at least two active redo files?
Also, assertion seems irrelevant.

Reply:
not present in latest commit.